model/foodEntry/index.js

```diff
@@ -16,23 +16,34 @@ import { db } from "../../firebase.js";
 
 import {
   collection,
+  query,
+  where,
   getDoc,
+  getDocs,
   setDoc,
   addDoc,
   updateDoc,
   deleteDoc,
   doc,
 } from "firebase/firestore";
 
-const foodEntryCollectionRef = collection(db, "foodEntries");
+const foodEntriesRef = collection(db, "foodEntries");
```

Comment:
What's the thinking behind using a class here? Pros/cons to a module?

Reply:
I thought about whether or not I wanted to be explicit in importing an entire class versus importing unique functions

Class:
import FoodEntryModel from "../.."
...
FoodEntryModel.getFoodEntry()

vs.

Module:
import { getFoodEntry } from "../.."
...
getFoodEntry()

My main concern was readability. When reading the class code, its very obvious that the function is a model function. In the module implementation, not so much.

Another solution would be to import a module like so:
import * as FoodEntryModel from "../.."

I read that this was frowned upon? Can't remember exactly why, but it probably has to do importing things you won't end up using.